pcdsdevices/interface.py

```diff
@@ -358,6 +364,21 @@ def subdevice_filter(info):
 
         return ophydobj_info(self, subdevice_filter=subdevice_filter)
 
+    def post_elog_status(self):
+        """
+        Post device status to the primary elog, if possible.
+        """
+        if has_elog:
```

Comment:
Some nitpicks:
* In general, it's better to exit early for bad conditions, at least so you can keep better indentation
```python
if not has_elog: 
    logger.info(...)
    return
```

* The `try/except ValueError` is wrapping 3 statements below; which does it apply to? 
* At least the `final_post = ...` assignment can be moved out of the try/except block, but not knowing about `get_primary_elog` or `elog.post()` I don't know where `ValueError` is expected to be raised